test/integration/features/basic.feature

```diff
@@ -113,6 +121,15 @@ Feature: Basic test
         Then stdout should contain "To login as a regular user, run 'oc login -u developer -p developer"
         And stdout should contain "To login as an admin, run 'oc login -u kubeadmin -p "
 
+    @darwin @linux @windows
+    Scenario: Monitoring stack check
+        Given checking that CRC is running
+        When login to the oc cluster succeeds
+        And executing "oc get pods -n openshift-monitoring" succeeds
+        Then stdout matches ".*cluster-monitoring-operator-[0-9a-z]{10}-[0-9a-z]{5}\ *2/2\ *Running.*"
```

Comment:
As per CI output the pod name doesn't have 10 char so better to use `.*cluster-monitoring-operator-[0-9a-z]+-[0-9a-z]+\ *2/2\ *Running.*` so we are not constraint to char number.

Reply:
Yes, just found out that max is 10, but could be less. Thanks! Also, what I don't understand is how come `oc login` succeeded when the test does not export `oc`'s location (`eval $(crc oc-env)`)... ci.centos failed on it but the prow one didn't.

Reply:
@jsliacan on e2e-crc the VM have oc in the path also :(